src/internal/cluster/secrets.go

```diff
@@ -15,16 +15,20 @@ import (
 	"github.com/defenseunicorns/zarf/src/pkg/message"
 )
 
+// DockerConfig contains the authentication information from the machine's docker config.
 type DockerConfig struct {
 	Auths DockerConfigEntry `json:"auths"`
 }
 
+// DockerConfigEntry contains a map of DockerConfigEntryWithAuth for a registry.
 type DockerConfigEntry map[string]DockerConfigEntryWithAuth
 
+// DockerConfigEntryWithAuth contains a dockker config authentication string.
 type DockerConfigEntryWithAuth struct {
 	Auth string `json:"auth"`
 }
 
+// GenerateRegistryPullCreds generates a secret containing the registry credentials
 func (c *Cluster) GenerateRegistryPullCreds(namespace, name string) (*corev1.Secret, error) {
 	message.Debugf("k8s.GenerateRegistryPullCreds(%s, %s)", namespace, name)
```

Comment:
Minor, pretty nothing remark:

Is the standard ending comments with a period or no?

I saw in some above changes there was just a period added to the end of a comment, so I am thinking the standard is to have sentence comments for `func` and `type`?

Reply:
Didn't see @Racer159 comment above.